src/smt/set_defaults.cpp

```diff
@@ -52,91 +52,91 @@ void setDefaults(LogicInfo& logic, bool isInternalSubsolver)
   // implied options
   if (options::debugCheckModels())
   {
-    options::checkModels.set(true);
+    Options::current().set(options::checkModels, true);
```

Comment:
Can we already just store a local variable `Options& opts = Options::current()` at the top of this file?

Eventually we will pass this to this method instead of calling Options::current() at all.